javascript/javascript2/week1/hyf-bay/main.js

```diff
@@ -0,0 +1,29 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+const main = document.querySelector('main');
+const ul = document.createElement('ul')
+main.appendChild(ul)
+
+function renderProducts(products) {
+
+  for (let i = 0; i < products.length; i++) {
+    const li = document.createElement('li')
+    const name = document.createElement('h1')
```

Comment:
Tiny thing:
I'd do `nameElement` or `nameHeader` to be more descriptive.

Reply:
👍🏻